### DISCUSSION TITLE
Errors opening ASP.NET - Boilerplate site fails to open

### DISCUSSION THREAD

**Discussion by xiecsuk** [Dotnet-Boxed/Templates#2] [Q&A]
Yesterday I installed the ASP.NET Boilerplate into my copy of Visual Studio 2013 Community edition and then successfully created a new website. I did not close VS after the installation. This morning, on opening VS, I am getting the following error messages:

SetSite failed for package [VisualSVN]
PowerShell Tools for Visual Studio Error - Failed to initialize PowerShell Tools

Is it possible to back out the Boilerplate to see if that has caused the problem or must I do a complete re-install of VS to remove it.

**Comment by RehanSaeed** [comment]
Boilerplate is just a very basic project template that gets added to the File->New Project menu. It can easily be uninstalled from the Tools->Extensions and Updates Menu.

However, it sounds like you have other problems not caused by the template which may have co-incidentally occurred when you installed or used the boilerplate template.

I would google those errors. A quick search throws up a few possible answers like <a href="http://forums.asp.net/t/1579603.aspx?Visual+SVN+fails+to+load+when+VS2010+is+started">this</a> one.

**Comment by xiecsuk** [comment]
Hi Rehan

I have uninstalled and reinstalled your template and everything seems to be working again. Fingers crosses.

Thanks
Roger Hampson

**Comment by RehanSaeed** [comment]
Thats good news. The installation went wrong maybe?
